### DISCUSSION TITLE
LDAP Issues (no sync?)

### DISCUSSION THREAD

**Discussion by Raven942** [kimai/kimai#2895] [On-Premise: Installation & Updates]
Hello,
I'm hoping for a little help or a small explanation as to what im doing wrong. I've attached an example of my LDAP configuration. Initially i was unable to login, however that was resolved from an update to php that did not install laminias LDAP. Once that was installed, i was able to login again. however after i created a new user, i set a basic password, but the Active Directory password would not work for the next login on the new user. i had to use the default password i set upon creation. i see nothing error wise in my logs while attempting to login, it simply states that the authentication is wrong.

Any insight?

kimai:
    user:
      registration: false
      password_reset: false
    permissions:
      roles:
          ROLE_USER: ['!password_own_profile']
          ROLE_TEAMLEAD: ['!password_own_profile']
          ROLE_ADMIN: ['!password_own_profile']
    ldap:
        activate: true
        connection:
            host: ldap://ip.ip.ip.ip
            port: 389
            username: admin@domain.local
            password: Password
            accountDomainName: domain.local
            accountDomainNameShort: domain
            accountFilterFormat: (&(objectClass=person)(sAMAccountName=%s))
        user:
            baseDn: dc=domain,dc=local
            filter: (&(objectClass=person))
            usernameAttribute: samaccountname
            attributesFilter: (objectClass=person)
            attributes:
                - { ldap_attr: mail, user_method: setEmail }
                - { ldap_attr: displayname, user_method: setAlias }
                - { ldap_attr: samaccountname, user_method: setUsername }
        role:
            baseDn: dc=domain,dc=local
            filter: (&(objectClass=group))
            groups:
                - { ldap_value: Leads, role: ROLE_TEAMLEAD }
                - { ldap_value: Domain Admins, role: ROLE_SUPER_ADMIN }
                - { ldap_value: Users, role: ROLE_USER }

**Comment by blattms** [comment]
If you are using ldap, then I would not create users in kimai. Kimai should create the user during the first login attempt.
